packages/evolution-backend/src/models/interviews.db.queries.ts

```diff
@@ -26,17 +30,51 @@ export interface InterviewSearchAttributes {
     username?: string | undefined;
     facebook: boolean;
     google: boolean;
+    surveyId?: number;
 }
 
+/** this will return the survey id or create a new survey in
+ * table sv_surveys if no existing survey shortname matches
+ * the project shortname
```

Comment:
Add a TODO that this should be done during server startup preparation so it is always available here and can be used from config

Reply:
added